### DISCUSSION TITLE
@cui/map vs @hurumap/core

### DISCUSSION THREAD

**Discussion by kilemensi** [CodeForAfrica/ui#5] [General]
Do we need to maintain 2 packages or is rolling all HURUmap components into a Commons UI sub-package a better way forward?

**Comment by KobbyMmo** [comment]
Do we have to maintain a package (npm) with this approach? or will they be projects under the mono repo. @kilemensi

**Reply by kilemensi** [reply]
Both: We'll need to maintain an npm package since some of the work we do will live outside the monorepo but either option will still live as packages under monorepo.
